### ISSUE TITLE
Semicolon at end of package declaration not reported, different error instead

### ISSUE THREAD

**Issue by gabrielittner** [shyiko/ktlint#59]
For

```kotlin
package com.example.foo;

data class Foo(val bar: String)
```

I get this: `com/example/foo/Foo.kt:1:25: Missing spacing after ";"` and ktlintFormat will add that space. Shouldn't the error be that there is a semicolon?

**Comment by shyiko** [comment]
Thank you, @gabrielittner! Fixed in 0.8.3 (Maven Central sync might take up to 25 minutes).
